ansible/group_vars/edusharing.yml

```diff
@@ -111,6 +111,8 @@ mirror_vocabularies: # list of vocabularies a local mirror should be created for
 #    target_directory: "/var/www/oer/vocabs-mirror"
 #  - url: "https://w3id.org/kim/hcrt/scheme.json"
 #    target_directory: "/var/www/oer/vocabs-mirror"
+#    isSorted: "true"
+#    variant: "true"
```

Comment:
Please also add a description for these two properties in the documentation of `mirror_vocabularies`. Please also note how the sorting is processed (sorted by xxx?)

Reply:
is ready